### PR TITLE
Implemented #653

### PR DESCRIPTION
**Pull request by shamikakumar**
#653 Replaced all alerts with modals

### CODE REVIEW

File: webui/src/components/auth/groups.js [hunk 1]
```diff
@@ -88,6 +91,7 @@ export const GroupsPage = connect(
                             <SyncIcon/>
                         </Button>
                     </ButtonToolbar>
+                    <ConfirmationModal show={show} onHide={handleClose} msg={deleteConfirmMsg} onConfirm={deleteSelectedGroups}></ConfirmationModal>
                 </div>
```
Comment (guy-har) [inline webui/src/components/auth/groups.js:95]: use self-closing form

File: webui/src/components/auth/policies.js [hunk 2]
```diff
@@ -85,6 +88,7 @@ export const PoliciesPage = connect(
                             <SyncIcon/>
                         </Button>
                     </ButtonToolbar>
+                    <ConfirmationModal show={show} onHide={handleClose} msg={confirmDeleteMsg} onConfirm={deleteSelectedPolicies}></ConfirmationModal>
```
Comment (guy-har) [inline webui/src/components/auth/policies.js:91]: use self-closing form

File: webui/src/components/auth/users.js [hunk 3]
```diff
@@ -252,12 +262,18 @@ const UserGroupPane = connect(
...
                             }}>Remove</Button>)
                         ]
                     }}
                 />
             </Form>
+            <ConfirmationModal show={show} onHide={handleClose} msg={detachConfirmMsg} 
+                onConfirm={() => {
+                    detachGroupFn(entityId);
+                }}>     
+            </ConfirmationModal>
```
Comment (guy-har) [inline webui/src/components/auth/users.js:276]: use self-closing form

**Review by guy-har — COMMENTED** [review]
@shamikakumar thanks for contributing, looks great!

one small request,
can you please use the self-closing form in the relevant tags

**Comment by shamikakumar** [comment]
@guy-har done!

**Comment by guy-har** [comment]
@shamikakumar Thanks,
I will approve and merge once you solve the issue commented by the CLAassistant above.
If you need any help, let me know.

**Comment by shamikakumar** [comment]
@guy-har I did sign the CLA and clicked on refresh not sure why it isnt updating ?
Screenshot 2020-10-07 at 12 09 48 PM

**Comment by arielshaqed** [comment]
Hi @shamikakumar 

I think this message says you might be using the wrong email address...  I'm clicking "recheck", otherwise please try to "add the email address used for this commit to your account"...

> [CLA assistant check](https://cla-assistant.io/treeverse/lakeFS?pullRequest=767)
> Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/treeverse/lakeFS?pullRequest=767) before we can accept your contribution.
> **Shamika Kumar** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/treeverse/lakeFS?pullRequest=767) it.

**Comment by arielshaqed** [comment]
Got it, @shamikakumar !  Your  commits are by `shamika_kumar@Shamikas-MacBook-Pro.local`, I assume that this is the local user on your Mac.  Obviously you didn't sign the CLA with that email.

In any case you should configure your _real_ email address on your local machine.  (AFAICT this is a specific bug on MacBooks, you always get a useless email address on your account, _not_ your fault.)  Do [this](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address), at least the steps you need to perform on your local `git`.  Then https://gist.github.com/trey/9588090 to edit the commit email address on each of the email messages, and then `git push --force-with-lease` to tell GitHub to look at the new rewritten history.

Hopefully this will work.  Feel free to contact me directly (e.g. on our Slack) or right here if this doesn't work.

**Comment by shamikakumar** [comment]
Thanks for the help @arielshaqed .

**Review by guy-har — APPROVED** [review]

**Comment by guy-har** [comment]
Thank you @shamikakumar, merged!

**Pull request merged**